Makefile

```diff
@@ -1,9 +1,28 @@
 default: base
 
 node_modules: package.json
-	npm install --legacy-peer-deps
+	npm clean-install --legacy-peer-deps
 	touch node_modules
 
+.PHONY: test-oidc
+test-oidc: node_modules
+	# TODO remove shellcheck call - not available/useful/etc. outside dev(?)
+	shellcheck ./oidc-tester/scripts/docker-start.sh
+	cd oidc-tester && \
+	docker compose down && \
+	docker compose build --build-arg CACHEBUST=$$RANDOM$$(date +%s) && \
+	docker compose up --exit-code-from odk-central-oidc-tester
+
+.PHONY: dev-oidc
+dev-oidc: base
+	NODE_CONFIG_ENV=oidc-development npx nodemon --watch lib --watch config lib/bin/run-server.js
```

Comment:
Can we add nodemon to dev dependencies and use npm here

Reply:
`nodemon` is already a dev dependency.  How might `npm` be used here?